send-custom-metrics/action.yaml

```diff
@@ -0,0 +1,23 @@
+name: Send Custom Metrics
+description: Sends custom metrics to an endpoint
+
+inputs:
+  METRICS_ENDPOINT_URL:
+    description: The metrics endpoint URL
+    required: true
+  BEARER_TOKEN:
+    description: The authorization token for the endpoint
+    required: true
+  data:
```

Comment:
can we do some additional validation on the `data`, and maybe have an option to take in a file in prometheus exposition format? https://github.com/prometheus/docs/blob/main/content/docs/instrumenting/exposition_formats.md#text-format-example

Separately, is there a way to send batch metrics?

Reply:
I updated this to add some quick validation for the metrics (thanks chatgpt)

@sionescu How many metrics do you think you'll be emitting from a workflow?  I prefer to keep things simple for now unless we have a use case in mind for batch metrics / files. We can add a version tag and add support for files later if they're needed

Reply:
@aluon Let's pause this for the moment, because sending a single data point is definitely not enough. I'll look into it.